LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -98,10 +101,21 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
             // ABP device does not reset the Fcnt so in relax mode we should reset for 0 (LMIC based) or 1
             var isFrameCounterFromNewlyStartedDevice = await DetermineIfFramecounterIsFromNewlyStartedDeviceAsync(loRaDevice, payloadFcntAdjusted, frameCounterStrategy);
```

Comment:
Did you consider the case where we have an ABP reset? As far as I read this then if we have a reset, all of the messages (from each concentrator), would reset the fcnt. Meaning that if we processed a message before from another concentrator, the fcnt we stored, would get reset and we would go to the function multiple times to perform the reset? What was the reason to add the deduplication after this?

Reply:
Changed in 745b009fa1dbc2b800b495f4700a523d6addeedb to reset only when not a duplicate.